templateutils/templateutils.go

```diff
@@ -277,6 +296,54 @@ func tablex(obj interface{}, minWidth, tabWidth int, userHeadings ...string) str
 	return createTable(obj, minWidth, tabWidth, headings)
 }
 
+func cols(obj interface{}, fields ...string) interface{} {
+	assertCollectionOfStructs(obj)
+	if len(fields) == 0 {
+		panic("at least one column name must be specified")
+	}
+
+	var newFields []reflect.StructField
+	var indicies []int
+	styp := reflect.TypeOf(obj).Elem()
+	for i := 0; i < styp.NumField(); i++ {
+		field := styp.Field(i)
+		if field.PkgPath != "" || ignoreKind(field.Type.Kind()) {
+			continue
+		}
+
+		var j int
+		for j = 0; j < len(fields); j++ {
+			if fields[j] == field.Name {
+				break
+			}
+		}
+		if j == len(fields) {
+			continue
+		}
+
+		indicies = append(indicies, i)
+		newFields = append(newFields, field)
+	}
+
+	if len(indicies) != len(fields) {
+		panic("not all column names are valid")
```

Comment:
Good question but this is already handled as we iterate through the fields of the structure and simply check whether these fields are present in the list of arguments passed to the function.  Thus if you call {{cols . "Name" "Name"}}, len(indicies) will be 1 and len(fields) will be two, and we'll error out.